data/json/items/tool_armor.json

```diff
@@ -1271,6 +1271,26 @@
     "flags": [ "BELTED", "FRAGILE", "ALLOWS_NATURAL_ATTACKS", "WATER_FRIENDLY", "OVERSIZE" ],
     "magazines": [ [ "battery", [ "light_minus_battery_cell", "light_minus_atomic_battery_cell" ] ] ]
   },
+  {
+    "id": "stepladder",
+    "type": "TOOL_ARMOR",
+    "name": "stepladder",
+    "description": "This is a wooden stepladder.  Use it to set it down.",
+    "weight": 20000,
+    "volume": 200,
+    "price": 2000,
+    "to_hit": -3,
+    "bashing": 12,
+    "material": "wood",
+    "symbol": "H",
+    "color": "brown",
+    "covers": [ "ARM_EITHER", "HAND_EITHER", "TORSO" ],
+    "coverage": 20,
+    "encumbrance": 120,
+    "material_thickness": 1,
+    "use_action": "LADDER",
+    "flags": [ "OVERSIZE", "RESTRICT_HANDS", "WATER_FRIENDLY" ]
```

Comment:
Needs "BELTED" flag here too, so that the stepladder is layered over all other worn items.